src/dev/build/lib/platform.js

```diff
@@ -17,22 +17,26 @@
  * under the License.
  */
 
-export function createPlatform(name) {
+export function createPlatform(name, architecture, buildName) {
   return new (class Platform {
     getName() {
       return name;
```

Comment:
So this returns `win32` now? That doesn't cause issues in a bunch of places?

Reply:
This change aligns with what is returned by [process.platform](https://nodejs.org/dist/latest-v10.x/docs/api/process.html#process_process_platform), removing the need for [mappings](https://github.com/elastic/kibana/pull/69165/files#diff-6b97e51ead00359fda5baa0e545c4a78L142) of any sort.